frontend/app/views/fragments/page/elevatedWhySupportBody.scala.html

```diff
@@ -0,0 +1,25 @@
+@import model.OrientatedImages
+
+@( image: OrientatedImages )
+
+<div class="l-constrained elevated-why-support-body">
+    <div class="elevated-why-support-body__title u-content-width--left">
+        <h1 class="elevated-why-support-body__title__h1">Why do we need</h1>
```

Comment:
Same points as above, regarding splitting the `<h1>` with a line break, and coupling class names to tag types.